ozaria/site/components/interactive/PageInteractive/draggableOrdering/index.vue

```diff
@@ -72,34 +136,53 @@
     :art-url="artUrl"
   >
     <div class="draggable-ordering-content">
-      <draggable
-        :list="promptSlots"
-        class="slots-container prompt-slots"
-        ghost-class="ghost-slot"
-        tag="ul"
-        :force-fallback="true"
-        fallback-class="dragging-slot"
-      >
-        <li
-          v-for="prompt in promptSlots"
-          :key="prompt.id"
-          :class="{ 'prompt': true, 'monospaced': (prompt.textStyleCode === true) }"
+      <div class="draggable-ordering-lists">
+        <draggable
+          :list="promptSlots"
+          class="slots-container prompt-slots"
+          ghost-class="ghost-slot"
+          tag="ul"
+          :force-fallback="true"
+          fallback-class="dragging-slot"
+        >
+          <li
+            v-for="prompt in promptSlots"
+            :key="prompt.id"
+            :class="{ 'prompt': true, 'monospaced': (prompt.textStyleCode === true) }"
+          >
+            {{ prompt.text }}
+          </li>
+        </draggable>
+
+        <ul
+          class="slots-container"
         >
-          {{ prompt.text }}
-        </li>
-      </draggable>
+          <li
+            v-for="(label, index) in labels"
+            :key="index"
+            :class="{ 'prompt-label': true, 'monospaced': (label.textStyleCode === true) }"
+          >
+            {{ label.text }}
+          </li>
+        </ul>
+      </div>
+
+      <base-button
+        class="submit"
+        :on-click="submitSolution"
+        :enabled="submitEnabled"
+      >
+        {{ $t('common.submit') }}
+      </base-button>
 
-      <ul
-        class="slots-container"
+      <modal-interactive
+        v-if="showModal"
+        @close="closeModal"
       >
-        <li
-          v-for="(label, index) in labels"
-          :key="index"
-          :class="{ 'prompt-label': true, 'monospaced': (label.textStyleCode === true) }"
-        >
-          {{ label.text }}
-        </li>
-      </ul>
+        <template v-slot:body>
+          <h1>{{ solutionCorrect ? "Did it!" : "Try Again!" }}</h1>
```

Comment:
Can use i18n?

Reply:
Same for all 3 types of interactives

Reply:
These are placeholders for now and will be resolved in UX spec phase